tests/test_snmp_notification_receiver.rb

```diff
@@ -0,0 +1,137 @@
+#
+# Minitest for SnmpNotificationReceiver class
+#
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/snmp_notification_receiver'
+
+# TestSnmpNotificationReceiver - Minitest for SnmpNotificationReceiver
+# node utility.
+class TestSnmpNotificationReceiver < CiscoTestCase
+  def setup
+    # setup runs at the beginning of each test
+    super
+    no_snmpnotificationreceiver
+    config('vrf context red')
+  end
+
+  def teardown
+    # teardown runs at the end of each test
+    no_snmpnotificationreceiver
+    config('no vrf context red')
+    super
+  end
+
+  def no_snmpnotificationreceiver
+    # Turn the feature off for a clean test.
+    config('no snmp-server host 4.5.6.7 informs version 3 priv ab udp-port 45',
+           'no snmp-server host 8.9.10.11 traps version 3 auth cd udp-port 46')
+  end
+
+  # TESTS
+
+  def test_snmpnotificationreceiver_create_destroy_single
```

Comment:
The convention we are trying to move towards is to not include the class name (`snmpnotificationreceiver`) in the test method name as it's redundant with the test class name (`TestSnmpNotificationReceiver::test_snmpnotificationreceiver_create_destroy_single`). Can you remove throughout?